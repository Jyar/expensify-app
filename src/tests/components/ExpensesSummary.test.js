import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary }  from '../../components/ExpensesSummary';

test('should return 1 expense if no 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount = {1} expensesTotal={235} />);
    expect(wrapper).toMatchSnapshot();
});

test('should return 2 expenses if multiple expenses', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount = {2} expensesTotal={235} />);
    expect(wrapper).toMatchSnapshot();
});


