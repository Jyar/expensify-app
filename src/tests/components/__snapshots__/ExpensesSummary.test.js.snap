// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return 1 expense if no 1 expense 1`] = `
<div
  className="page-header"
>
  <div
    className="content-container"
  >
    <h1
      className="page-header__title"
    >
      Viewing 
      <span>
         
        1
      </span>
       
      expense
       totaling 
      <span>
        $2.35
         
      </span>
    </h1>
    <div
      className="page-header__actions"
    >
      <Link
        className="button"
        replace={false}
        to="/create"
      >
        Create Expense
      </Link>
    </div>
  </div>
</div>
`;

exports[`should return 2 expenses if multiple expenses 1`] = `
<div
  className="page-header"
>
  <div
    className="content-container"
  >
    <h1
      className="page-header__title"
    >
      Viewing 
      <span>
         
        2
      </span>
       
      expenses
       totaling 
      <span>
        $2.35
         
      </span>
    </h1>
    <div
      className="page-header__actions"
    >
      <Link
        className="button"
        replace={false}
        to="/create"
      >
        Create Expense
      </Link>
    </div>
  </div>
</div>
`;
